app/javascript/components/widgets/widgets/forest-change/fires-alerts/selectors.js

```diff
@@ -0,0 +1,296 @@
+import { createSelector } from 'reselect';
```

Comment:
How much has this file deviated from the GLADs? Could we map the fires data to the same structure in the action and then make this a helper? Or even just reuse it from GLADs?

Reply:
Yes, I think so - I just wanted to start from scratch first to find if there are many deviations from GLAD.